@shared/utils/gqlClient.ts

```diff
@@ -14,6 +14,7 @@ const client = new GraphQLClient(process.env.NEXT_PUBLIC_GALACTUS_BASE_URL, {
     'X-ENABLE-NOUNS': 'true',
   }),
   mode: 'no-cors',
+  cache: 'no-store',
```

Comment:
any details on this?

Reply:
Something changed here in your PR from the merge... Can't recall if you had removed it or not. I'll pull this out.